libs/stringtable/src/rapify.rs

```diff
@@ -137,7 +137,11 @@ pub fn rapify(project: &Project) -> Option<XmlbLayout> {
                 i32::try_from(translation.phrases.len()).expect("overflow"),
             );
             for phrase in &translation.phrases {
-                write_string(&mut translation_buffer, phrase);
+                let unescaped = quick_xml::escape::unescape(phrase.as_str());
+                if unescaped.is_err() {
+                    error!("failed to unescape {}", phrase);
```

Comment:
Should either fail the build, prevent using a bin stringtable, or should be a warning